runtime/src/append_vec.rs

```diff
@@ -13,11 +13,12 @@ use std::{
     sync::Mutex,
 };
 
-//Data is aligned at the next 64 byte offset. Without alignment loading the memory may
```

Comment:
I'm fairly certain `64 byte offset` is wrong description; it should be `8 byte offset` or `64 bit offset` if you prefer `bits`. Padding at 64 byte boundary would be too wasteful. I've never heard of such architecture. Also, the macro impl doesn't look like actualy aligning with 64 byte, too.

Reply:
Yea 64-byte in the description is wrong, but some vector instructions like vmovapd can require 64-byte alignment for avx-512 moves:
https://www.felixcloutier.com/x86/movapd

Reply:
Of course compilers will probably always emit the unaligned-tolerant versions of those instructions.

Reply:
> avx-512 moves

Oh, the mighty 512 bits! Yeah, 64-byte alignment will be warranted in some special cases! Thanks for the tip!